docs/user-guide/src/bmo/automatic_secure_boot.md

```diff
@@ -0,0 +1,30 @@
+# Automatic secure boot
+
+The automatic secure boot feature allows enabling and disabling UEFI (Unified Extensible Firmware Interface) secure boot when provisioning a host. This feature requires supported hardware and compatible OS image. The current hardwares that support enabling UEFI secure boot are `iLO driver`, `iRMC driver` and `Redfish driver`.
```

Comment:
nit: not sure why use ticks around drivers, maybe you meant `iLO`, `iRMC` and `Redfish` drivers?